.changes/unreleased/Breaking Changes-20221206-142700.yaml

```
@@ -0,0 +1,7 @@
+kind: Breaking Changes
+body: 'from_description method returns properly types with specified time zone '
```

`adapter.get_columns_in_relation`.
```suggestion
body: '`adapter.get_columns_in_relation` now properly returns the type of timestamp and time with timezone columns. Before this fix these were incorrectly returned as timestamp and time respectively.'
```

`from_description` is not a method dbt users will know. the method they will know is `adapter.get_columns_in_relation`.

Also specified time zone seems misleading as it is only a fix related to the type of a column. The time zone is not specified there.

---

changed